infrastructure/food_oasis_app_stack/main.tf

```diff
@@ -1,43 +1,43 @@
 provider "aws" {
-  region  = var.region
+  region  = module.networked_rds.region
 }
-# module "networked_rds" {
-#   source = "../../../terraform_files/modules/networked_rds"
+module "networked_rds" {
+  source = "../networked_rds"
```

Comment:
Please use a source path to github instead of your local filesystem. If you have some changes here on your local please PR them to the upstream repo
[see docs](https://www.terraform.io/docs/modules/sources.html#github) for example

```hcl
module "networked_rds" {
  source = "github.com/100Automations/terraform-aws-postgres-vpc?ref=tags/v0.3.1"
  # etc
}
```

Reply:
okay